File: 11866/kotlin/boj_11866.kt
fun main() {
    val (n, k) = readln().split(" ").map { it.toInt() }
    val josephus_list = mutableListOf<Int>()
    val range = MutableList(n) { it + 1 }

    var idx = 0
    while (range.isNotEmpty()) {
        idx = (idx + k - 1) % range.size
        josephus_list.add(range[idx])
        range.removeAt(idx)
    }
    println("<${josephus_list.joinToString(", ")}>")
}

File: 11866/python/boj_11866.py
import sys

input = sys.stdin.readline

n, k = map(int, input().split())
josephus_list = []

rng = list(range(1, n+1))
idx = 0
while rng:
    idx = (idx + k - 1) % len(rng)
    josephus_list.append(rng.pop(idx))

print(f"<{', '.join(map(str, josephus_list))}>")
